cmd/main.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"github.com/getsentry/sentry-go"
+	"log"
+	"pkg.world.dev/world-cli/cmd/world/root"
+	"time"
+)
+
+func main() {
+	// Sentry initialization
+	err := sentry.Init(sentry.ClientOptions{
+		Dsn:                "{Input you DSN here}",
```

Comment:
How is this field supposed to be used? Shouldn't this be set via a config file?

Reply:
it will be placed when the apps will be builded (go build)
the value should be secret because if user have it, they can spam to our sentry server.

maybe i can make sentry automatically deactivated when DSN doesn't filled. 
what do you think?

Reply:
Yes, I agree with disabling sentry reporting if this is empty.

Reply:
updated